scripts/ubuntu1604/lineage.sh

```diff
@@ -456,17 +413,17 @@ echo
 echo NAME=\$NAME
 echo EMAIL=\$EMAIL
 echo
-echo "Override the above environment variables in your Vagrantfile to alter the build configuration."
+echo "Override the above environment variables to alter the build configuration."
 echo
 echo
 sleep 10
 
 # Setup the branch and enable the distributed cache.
 export USE_CCACHE=1
+export CCACHE_DIR="\$HOME/cache"
+export CCACHE_COMPRESS=1
 export TMPDIR="\$HOME/temp"
-export ANDROID_CCACHE_SIZE="20G"
-export ANDROID_CCACHE_DIR="\$HOME/cache"
-export PROCESSOR_COUNT=`cat /proc/cpuinfo  | grep processor | wc -l`
+export PROCESSOR_COUNT=\`cat /proc/cpuinfo | grep processor | wc -l\`
```

Comment:
[`nproc`](https://man7.org/linux/man-pages/man1/nproc.1.html) - print the number of processing units available.
```suggestion
export PROCESSOR_COUNT=\$(nproc)
```